<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="2" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="11067" started-at="2021-06-22T19:12:38Z" finished-at="2021-06-22T19:12:49Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="11067" started-at="2021-06-22T19:12:38Z" finished-at="2021-06-22T19:12:49Z">
      <class name="testCases.POST_Or_Create_A_Product">
        <test-method status="PASS" signature="create_A_product()[pri:0, instance:testCases.POST_Or_Create_A_Product@12b0404f]" name="create_A_product" duration-ms="5207" started-at="2021-06-22T19:12:38Z" finished-at="2021-06-22T19:12:44Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- create_A_product -->
        <test-method status="FAIL" signature="read_A_product()[pri:0, instance:testCases.POST_Or_Create_A_Product@12b0404f]" name="read_A_product" duration-ms="5748" started-at="2021-06-22T19:12:44Z" finished-at="2021-06-22T19:12:49Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[The following asserts failed:
Procuct Id is not Matching! expected [1808] but found [null],
Procuct Name is not Matching! expected [Automation Basic 1.0] but found [null],
Procuct Price is not Matching! expected [100] but found [null]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: The following asserts failed:
Procuct Id is not Matching! expected [1808] but found [null],
Procuct Name is not Matching! expected [Automation Basic 1.0] but found [null],
Procuct Price is not Matching! expected [100] but found [null]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
at testCases.POST_Or_Create_A_Product.read_A_product(POST_Or_Create_A_Product.java:83)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- read_A_product -->
      </class> <!-- testCases.POST_Or_Create_A_Product -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
